introduction.md

```diff
@@ -0,0 +1 @@
+Hi!  My name is Zack and I am super excited to be here.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.